spreadsheet/workbook.go

```diff
@@ -779,3 +794,21 @@ func (wb *Workbook) RemoveCalcChain() {
 func (wb *Workbook) GetFilename() string {
 	return wb.filename
 }
+
+// GetOrCreateCustomProperties returns the custom properties of the document (and if they not exist yet, creating them first)
+func (wb *Workbook) GetOrCreateCustomProperties() common.CustomProperties {
```

Comment:
Why were changes regarding custom properties needed for playground?  Forgot

Reply:
As far as I remember, a similar function was already in document and I decided to add it to spreadsheet to test do custom properties work or not because they showed error in the playground